src/pages/staking-hub/wrapper.tsx

```
@@ -184,44 +202,179 @@ function WrapperPage() {
 
+  // xHOPR -> wxHOPR if handlerData set
+  const { data: xHOPR_to_wxHOPR_config, refetch: refetch1 } = useSimulateContract({
+    address: xHOPR_TOKEN_SMART_CONTRACT_ADDRESS,
+    abi: web3.wrapperABI,
+    functionName: 'transferAndCall',
+    args: [address, parseUnits(xhoprValue as NumberLiteral, 18), '0x'],
+  });
+
+  // xHOPR -> wxHOPR multicall if handlerData not set
+  const addressWithout0x = address?.replace('0x','').toLocaleLowerCase() || '';
+  const setInterfaceImplementer = `29965a1d000000000000000000000000${addressWithout0x}b281fc8c12954d22544db45de3159a39272895b169a852b314f9cc762e44c53b000000000000000000000000e530e2f9decf24d7d42f011f54f1e9f8001e7619`;
```

ESLint

[error] 'setInterfaceImplementer' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)